src/main/java/com/product/controller/ProductController.java

```diff
@@ -30,10 +31,13 @@
 
 	@Autowired
 	private ProductReviewService productReviewService;
-
+	
+	
 	@GetMapping("/")
-	public ResponseEntity<List<Product>> getProduct() {
-		return new ResponseEntity<List<Product>>(productSrervice.getAllProducts(), HttpStatus.OK);
+	public ResponseEntity<List<Review>> getProduct() {
+		//List<ProductDTO> dtos=new ArrayList<ProductDTO>();
+		List<Review> reviewList=productReviewService.getProductReviews(1);
```

Comment:
getProductReviews(1) - why are you hardcoding 1?

Reply:
sorry, i forgot to remove, after the test.